sources/updater/README.md

```diff
@@ -5,6 +5,7 @@ This document describes the Bottlerocket update system and its components, namel
 - updog: update client that interfaces with a TUF repository to find and apply updates
 - signpost: helper tool to update partition priority flags
 - Bottlerocket update operator (brupop): an optional component that coordinates node updates with the rest of the cluster
+- Bottlerocket update API: a set of API endpoints that facilitates stateful updates
```

Comment:
```suggestion
- Bottlerocket update API: a set of API calls for checking and starting system updates
```

Reply:
nit: maybe the start of each line in this table of contents should be a link to the section?